internal/service/eventtrigger/resource_event_trigger_test.go

```diff
@@ -52,33 +52,79 @@ func TestAccEventTrigger_basic(t *testing.T) {
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck:                 func() { acc.PreCheckBasic(t) },
 		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
-		CheckDestroy:             testAccCheckMongoDBAtlasEventTriggerDestroy,
+		CheckDestroy:             checkDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccMongoDBAtlasEventTriggerDatabaseConfigDatabase(projectID, appID, `"INSERT", "UPDATE"`, &event, false, false),
+				Config: configDatabaseTrigger(projectID, appID, `"INSERT", "UPDATE"`, &event, false, false),
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckMongoDBAtlasEventTriggerExists(resourceName, &eventResp),
+					checkExists(resourceName, &eventResp),
 					resource.TestCheckResourceAttr(resourceName, "project_id", projectID),
 				),
 			},
 			{
-				Config: testAccMongoDBAtlasEventTriggerDatabaseConfigDatabase(projectID, appID, `"INSERT", "UPDATE", "DELETE"`, &eventUpdated, true, true),
+				Config: configDatabaseTrigger(projectID, appID, `"INSERT", "UPDATE", "DELETE"`, &eventUpdated, true, true),
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckMongoDBAtlasEventTriggerExists(resourceName, &eventResp),
+					checkExists(resourceName, &eventResp),
 					resource.TestCheckResourceAttr(resourceName, "project_id", projectID),
 				),
 			},
 			{
 				ResourceName:      resourceName,
-				ImportStateIdFunc: testAccCheckMongoDBAtlasEventTriggerImportStateIDFunc(resourceName),
+				ImportStateIdFunc: importStateIDFunc(resourceName),
 				ImportState:       true,
 				ImportStateVerify: true,
 			},
 		},
 	})
 }
 
-func TestAccEventTriggerDatabase_eventProccesor(t *testing.T) {
+func TestAccEventTrigger_databaseNoCollection(t *testing.T) {
+	acc.SkipTestForCI(t)
```

Comment:
is it hard to make it run in CI?

Reply:
It does have some complexity, details of required setup are detailed [here](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/cfn-resources/trigger/test#manual-qa). We would have to use the realm SDK to create some app service specific resources.